runtime/src/lib.rs

```diff
@@ -62,7 +62,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_version: 34,
```

Comment:
I believe we also need to bump the `spec_version`, right?

Reply:
Yes we have to do that between two releases, ideally when the first change in the runtime logic is introduced after a release. The previous release had a `spec_version` that's 1 version lower, so we're good to go :)

Reply:
Ok, I see! Thanks!